cluster/manifests/spot-node-rescheduler/cronjob.yaml

```diff
@@ -0,0 +1,35 @@
+{{ if eq .ConfigItems.spot_node_rescheduler "true" }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: node-spot-rescheduler
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: node-spot-rescheduler
+spec:
+  schedule: "*/5 * * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: 1800
+      backoffLimit: 1
+      template:
+        metadata:
+          labels:
+            application: kubernetes
+            component: node-spot-rescheduler
+        spec:
+          serviceAccountName: node-spot-rescheduler
+          restartPolicy: Never
+          containers:
+          - name: node-spot-rescheduler
+            image: container-registry.zalando.net/teapot/spot-node-rescheduler:main-1
+            resources:
+              limits:
+                cpu: 50m
+                memory: 384Mi
+              requests:
+                cpu: 50m
+                memory: 384Mi
```

Comment:
It's good practice to define these with a config-item such that we can increase per cluster if needed. We don't know ahead of time how much it will consume so better to make it easy to change. See other manifests for examples.